src/styles.css

```diff
@@ -175,3 +176,43 @@ footer {
   position: relative;
   right: 20px;
 }
+
+form {
+  position: relative;
+  top: 60px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 600px;
+  margin: auto;
+}
+
+form h4 {
+  display: flex;
+  justify-content: center;
+  margin-top: 5px;
+  font-size: 30px;
+}
+
+form input {
+  padding: 10px;
+  margin: 20px 5px 5px 5px;
+  width: 285px;
+}
+
+form textarea {
+  padding: 10px;
+  margin: 5px;
+  outline: none;
+  border: 1px solid black;
+  resize: none;
+  width: 420px;
+}
+
+.popup-btn {
+  padding: 10px;
+  cursor: pointer;
+  width: 107px;
+  margin-left: 4px;
+  margin-bottom: 5px;
+}
```

Comment:
[Suggestion] that would be really good if you apply some Hovers with change background color.